app/controllers/charges_controller.rb

```diff
@@ -19,7 +19,7 @@ def create
 
     if charge.paid?
       message = 'Your transaction was successful!'
-      redirect_to cart_path, notice: message
+      redirect_to charges_path, notice: message
```

Comment:
All of this seems out of scope. Why do you choose to keep this controller around?

Reply:
Yes it is, we are taking it out from this PR and making a separate PR for it 👍